### ISSUE TITLE
[A11Y] Android: Unlabeled image and icons are present on the Article template page

### ISSUE THREAD

**Issue by gkhadke** [ampproject/ampstart#381]
**ENVIRONMENT:**

OS (number): N (7.1.2)
Build number: N2G47H
Form factor (Smartphone / tablet):  Nexus 6P
Browser: Chrome Dev 59.0.3056.4

**Pre-requisite:**
1. Talkback is ON

**STEPS TO REPRODUCE:**

1. Open [https://ampstart.com](https://ampstart.com)
2. Linearly navigate to the “Article” template and double tap on “View Template” link
3. Try to linearly navigate through the section where leopard image is present.

**ACTUAL RESULTS:**

Unlabeled image and image icons are present in that section and talkback reads out some text (most likely the filename of image/icons)

**EXPECTED RESULTS:**

If images are purely decorative and doesn’t convey any info, then alt tag should be “null”, so screen reader can skip them. If Images provide useful information then, appropriate label should be used for the images along with description (Note: Description should be used only if it’s required.)

**ATTACHMENT(S):**

<img width="1440" alt="unlabeled_image" src="https://cloud.githubusercontent.com/assets/26986019/24821087/613a369c-1ba1-11e7-8a15-3ec768a5d43c.png">
<img width="1440" alt="unlabeled_imageicon1" src="https://cloud.githubusercontent.com/assets/26986019/24821085/6136f892-1ba1-11e7-9e8e-1afca16388cf.png">
<img width="1440" alt="unlabeled_imageicon2" src="https://cloud.githubusercontent.com/assets/26986019/24821086/6139f8da-1ba1-11e7-81b6-221ebc2638c7.png">

**Comment by ericlindley-g** [comment]
Good catch — though this is out of our control, since this is inside an Instagram embed.
